### ISSUE TITLE
Trivial code hangs the analyzer

### ISSUE THREAD

**Issue by newobj** [ccontrols/structured-types#5]
```
export class Test {
  make(gen: Object) { return 0 }
}
```

Analyzing this code will hang the analyzer forever (I wonder if it's an infinite loop somewhere resolving Object? just guessing?)

'any' works fine:

```
export class Test {
  make(gen: any) { return 0 }
}
```

yields

```
{
  Test: {
    name: 'Test',
    kind: 13,
    properties: [
      {
        name: 'make',
        kind: 11,
        parameters: [ { name: 'gen', kind: 17 } ],
        returns: { kind: 2 }
      }
    ]
  }
}
```

**Comment by atanasster** [comment]
spot on with your guess, fixing it now

**Comment by atanasster** [comment]
What do you think - we should map `Object` to `unknown`(my inclination) - or just to `object`?

**Comment by atanasster** [comment]
Fixed in `3.20.2`
Here is the playground: https://bit.ly/3lRoBbL (mapping to unknown for now but we can change that)

**Comment by newobj** [comment]
unknown seems fine to me! Thanks!
